### ISSUE TITLE
 timgmsoundfont doesn't find local source archive

### ISSUE THREAD

**Issue by humdingerb** [haikuports/haikuports#3565]
The TimGM soundfont's "source" archive is stored locally in a "source" folder alongside the recipe. Building fails with:
```
Downloading: file://source/timgmsoundfont-fixed.zip ...
Warning: Unable to fetch source from file://source/timgmsoundfont-fixed.zip
(error: source /Ports/haikuports/_x86_gcc2/haiku-data/timgmsoundfont/source/timgmsoundfont-fixed.zip
doesn't exist), trying next location.
```

I'd like to package another soundfont (30 MiB) which is hosted, unfortunately, at a dropbox. Should I go the same store-local-route as this TimGM recipe, as a dropbox link feels a bit volatile?

**Comment by pulkomandy** [comment]
I would rather avoid this. We could store things at HaikuArchives or so.
But I don't know if dropbox is more volatile than "some random developer webpage", after all.

**Comment by humdingerb** [comment]
In its license text, it says:
> If you plan to feature GeneralUser GS on your own website, please do not link directly to my download files.  Either link to my website, or provide your own local copy instead.

I'll try to contact the author and confirm we can use the dropbox link, or if we should host the file at e.g. HaikuArchives. I suppose, since the soundfont won't change much, our buildbots (and the few people building recipes manually) won't need to download the archive often?

**Comment by pulkomandy** [comment]
Yes, packaging into an hpkg counts as making a local copy, even if the process is automated. Also, we are not a website :)

**Comment by humdingerb** [comment]
Thanks for the feedback, PR coming up.
The ticket is still valid, I suppose, as building from a local (file://) archive doesn't seem to work.

> Also, we are not a website :)

That's for the lawyers to decide. :)

**Comment by waddlesplash** [comment]
Just use the ports-mirror then.
